vmdb/spec/models/ems_refresh/refreshers/openstack_refresher_rhos_havana_spec.rb

```diff
@@ -7,537 +7,562 @@
     @ems.update_authentication(:default => {:userid => "admin", :password => "password"})
   end
 
-  it "will perform a full refresh against RHOS Havana" do
-    2.times do  # Run twice to verify that a second run with existing data does not change anything
-      @ems.reload
-      # Caching OpenStack info between runs causes the tests to fail with:
-      #   VCR::Errors::UnusedHTTPInteractionError
-      # Reset the cache so HTTP interactions are the same between runs.
-      @ems.reset_openstack_handle
-
-      # We need VCR to match requests differently here because fog adds a dynamic
-      #   query param to avoid HTTP caching - ignore_awful_caching##########
-      #   https://github.com/fog/fog/blob/master/lib/fog/openstack/compute.rb#L308
-      VCR.use_cassette("#{described_class.name.underscore}_rhos_havana", :match_requests_on => [:method, :host, :path]) do
-        EmsRefresh.refresh(@ems)
+  context "with no errors collecting inventory" do
```

Comment:
To keep it simple, should we just leave this context alone, then add a separate context or a separate entire spec file to test for the errors during refresh?